lib/CompilerManager.js

```diff
@@ -143,13 +143,30 @@ class CompilerManager {
 
     /**
      * Get more complete compiler info on all compilers for debugging purposes
+     * @param {String[]} configNames list of all compilers (active and inactive)
      * @return {Object} a map of data keyed on compiler names
      */
-    getAllCompilerInfo() {
+    getAllCompilerInfo(configNames) {
```

Comment:
The compiler manager is responsible for managing active compilers only. Augmenting this method to tell us information about inactive compilers by passing in all compilers mixes concerns, which isn't ideal. I think it would be best to move the logic for incorporating inactive compilers into the result to the `if (reqPath === '${this.kevinApiPrefix}/compiler-info') {` conditional in `middleware.js`.

```js
const activeCompilerInfo = manager.getAllCompilerInfo();
this.configs.forEach(({ name }) => {
    if (activeCompilerInfo.compilers.hasOwnProperty(name)) {
        return;
    }
    activeCompilerInfo.compilers[name] = { status: NOT_BUILT }
});
```